index.js

```diff
@@ -0,0 +1,29 @@
+document.getElementById('hamburger').onclick = () => {
+  document.getElementById('navulitems').classList.add('hamburger');
+  document.getElementById('hamburger').classList.add('hamx');
+  document.getElementById('x').classList.remove('hamx');
+};
+
+document.getElementById('x').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l1').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l2').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
```

Comment:
- [ ] It was working properly so the first reviewer did not give the required changes but now it is not working please fix it. You have a horizontal scrolling problem please fix it on mobile screen.
### Error
![Screenshot 2023-01-04 175304](https://user-images.githubusercontent.com/81727988/210551983-4c29ee89-a662-49ab-9013-604449790a88.png)